### PR TITLE
Update reference to the current WT commandline

### PR DESCRIPTION
**Pull request by marjimgu**
The index.md is outdate indicating there is not commandline options but since versio XXXX there. And some are docummented in UsingCommandlineArguments.md. 

I also added reference to list to changes of the WT releases.

I removed the line. "None at this time. See issue [#607](https://github.com/microsoft/terminal/issues/607)"

## PR Checklist
* [ ] Closes #xxx
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

### CODE REVIEW

File: doc/user-docs/index.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ change. If you notice an error in the docs, please raise an issue. Or better yet
 
 To compile Windows Terminal yourself using the source code, follow the instructions in the [README](/README.md#developer-guidance).
 
+You check the WT [releases changes](https://github.com/microsoft/Terminal/releases).
```
Comment (jsoref) [inline doc/user-docs/index.md]: This doesn't make sense.

Comment (zadjii-msft) [inline doc/user-docs/index.md]: I'm not sure this line's totally necessary here. Kinda feels out of place here to me. At the very least, I'd reword it 
```suggestion
Please refer to the Windows Terminal [releases notes](https://github.com/microsoft/Terminal/releases) for the latest changes.
```

File: doc/user-docs/index.md [hunk 2]
```diff
@@ -25,7 +27,8 @@ NOTE: The default shell is PowerShell; you can change this using the _Running a
 
 ### Command line options
 
-None at this time. See issue [#607](https://github.com/microsoft/terminal/issues/607)
+Please check [Using the wt.exe Commandline](https://github.com/microsoft/terminal/blob/master/doc/user-docs/UsingCommandlineArguments.md)
```
Comment (zadjii-msft) [inline doc/user-docs/index.md]:
```suggestion
Please refer to [Using the wt.exe Commandline](https://github.com/microsoft/terminal/blob/master/doc/user-docs/UsingCommandlineArguments.md) for details on how to use the `wt.exe` commandline arguments.
```

**Comment by cinnamon-msft** [comment]
I agree with @zadjii-msft's recommendations. :)

**Comment by DHowett-MSFT** [comment]
Hey, thanks for the docs contribution! We're going to be revisiting our user docs soon, and in the interest of reducing file conflicts we're not taking on changes at this exact moment. I'll be happy to have you back when we've cleared that hurdle :smile:

**Comment by marjimgu** [comment]
Hello Dustin. Thanks for the feedback.
